UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/appsettings.json

```diff
@@ -106,5 +106,9 @@
   "PeriodicOutputServiceConfiguration": {
     "LargeMediaExchangeSetSizeInMB": 700,
     "LargeExchangeSetFolderName": "M0{0}X02"
+  },
+  "AioConfiguration": {
+    "AioEnabled": true,
```

Comment:
Would these values need to be configured within the environment libraries?

It is likely that we will initially need this set to true in Dev and QA but false in Live so that the features can be built and tested but switched off in the production environment.

Reply:
Absolutely, Martin. These values will certainly be included in the environment variables. Given that the branch will be merged into main and that a deployment may occur in the future, we can keep this false as the default value for the time being until we implement this in the library. Let us know your thoughts.

Reply:
Thanks - Happy if the value is false at the moment and moved into environment variables in due course.

Reply:
Resolved as AIOEnabled value set to false in appsettings